### ISSUE TITLE
Any tips for finetuning on private datasets?

### ISSUE THREAD

**Issue by alen-mask** [VDIGPKU/CBNet_caffe#16]
Hi @PKUbahuangliuhe ,

Great work! So fay as i know, cbnet is the sota on coco object detection tasks.
Do you have any plans to support finetuning on other datasets(different number of object classes) from your pretrained model?
Any tips or reference code here?

Many thanks!

**Comment by PKUbahuangliuhe** [comment]
> 
> 
> Hi @PKUbahuangliuhe ,
> 
> Great work! So fay as i know, cbnet is the sota on coco object detection tasks.
> Do you have any plans to support finetuning on other datasets(different number of object classes) from your pretrained model?
> Any tips or reference code here?
> 
> Many thanks!

you can convert the format of your data to COCO.Note that the number class should also be modified,and just train it.

**Comment by alen-mask** [comment]
> > Hi @PKUbahuangliuhe ,
> > Great work! So fay as i know, cbnet is the sota on coco object detection tasks.
> > Do you have any plans to support finetuning on other datasets(different number of object classes) from your pretrained model?
> > Any tips or reference code here?
> > Many thanks!
> 
> you can convert the format of your data to COCO.Note that the number class should also be modified,and just train it.

Thanks very much, guess i should read the training code again.
